roles/setup_sushy_tools/tasks/main.yml

```diff
@@ -84,7 +84,7 @@
 
     - name: Set X86_64 OVMF code path
       set_fact:
-        sushy_x86_64_ovmf_code_path: "/usr/share/edk2/ovmf/OVMF_CODE.secboot.fd"
+        sushy_x86_64_ovmf_code_path: "/usr/share/OVMF/OVMF_CODE.secboot.fd"
```

Comment:
This shouldn't be changed, the path you've changed it to gets stat-ed in the next task and replaced if its found.

Reply:
If we need the same path for sushy and the vm then we should duplicate/point the logic we have here in the VM role.

Reply:
I'll retry with removing this change, to see how it goes

Reply:
We definitely need this change in the OVMF code path, because it's failing when trying to boot the VM: https://www.distributed-ci.io/jobs/9c30d6e3-6dc3-4aa8-b0c9-ed34b1ebd1ec/jobStates?sort=date&task=f9f27ff9-1f3d-4218-a5a2-b6e6d37f4bbd, so I'll add a check for disabling secure boot in the sushy tools role too

Reply:
Now it's passing, with the new refactor of setup_sushy_tools: https://www.distributed-ci.io/jobs/ecb35cf2-d87e-4df4-ad1a-cc1f99f78f8f/jobStates?sort=date&task=43930d9d-a4c9-4209-9cb5-de850b000741